manifests/mirror/config.pp

```diff
@@ -19,15 +19,27 @@
     fail("Use of private class ${name} by ${caller_module_name}")
   }
 
-  ::kafka::consumer::config { 'consumer':
+  if $consumer_config['group.id'] == '' {
```

Comment:
It will throw an error message like ```fail('[Consumer] You need to specify a value for group.id')```. Otherwise the service ```kafka::mirror``` will not start and will complain about the missing parameter at ```/var/log/kafka/mirror.log```.